console-store/Clases/Product.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleStore.Clases
+{
+    internal class Product
```

Comment:
Usually c# developers make classes public if no specific requirements

Reply:
More details. Are we talking about `internal`?

Reply:
Yes, I am talking about `internal` accessor. In most cases, I would use `public` instead